src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -119,17 +121,16 @@ private void initService() {
     container.setVoteDAO(voteDAO);
     container.setMatchDAO(matchDAO);
     service = new DataAccessRequestService(counterService, container, dacService,
-        dataAccessRequestServiceDAO, userService);
+        dataAccessRequestServiceDAO, userService, institutionService);
   }
 
   @Test
   void testCreateDataAccessRequest_Update() {
     DataAccessRequest dar = generateDataAccessRequest();
     dar.setCollectionId(null);
     dar.addDatasetIds(List.of(1, 2, 3));
-    User user = new User(1, "email@test.org", "Display Name", new Date());
-    user.setLibraryCards(List.of(new LibraryCard()));
-    user.setEraCommonsId("eraCommonsId");
+    User user = createUserWithPrerequisites();
+    when(institutionService.findInstitutionForEmail(any())).thenReturn(user.getInstitution());
```

Comment:
Maybe this should be done in `generateDataAccessRequest()`? It looks like every caller to that method will need this mock, and this test user can be added as a field.